tests/vision/checks/performance/robustness_report_test.py

```diff
@@ -37,18 +35,17 @@ def test_mnist(mnist_dataset_train, trained_mnist, device):
     ]
     check = RobustnessReport(augmentations=augmentations)
     # Act
-    result = check.run(mnist_dataset_train, trained_mnist,
-                       prediction_formatter=ClassificationPredictionFormatter(mnist_prediction_formatter),
-                       device=device)
+    result = check.run(mnist_dataset_train, trained_mnist, device=device)
     # Assert
+    print(result.value)
     assert_that(result.value, has_entries({
         'RandomBrightnessContrast': has_entries({
             'Precision': has_entries(score=close_to(0.984, 0.001), diff=close_to(-0.001, 0.001)),
-            'Recall': has_entries(score=close_to(0.986, 0.001), diff=close_to(-0.001, 0.001))
+            'Recall': has_entries(score=close_to(0.986, 0.001), diff=close_to(-0.0003, 0.001))
```

Comment:
if we moved the set_seeds before the metrics, doesn't the results should remain the same?

Reply:
no, because it wasn't changed just before before the refactor